### DISCUSSION TITLE
Attempt in Ventura

### DISCUSSION THREAD

**Discussion by agala68** [ivangabriele/clamav-desktop#69] [Questions & Answers]
Hi, I just did an attempt torun in Ventura, and I get the app compiled, but running it it seems thong, the only error I can see in building it, is this one:

TypeError: Cannot set property mark of #<Object> which has only a getter

Any idea?
Is it too old to run in 13.x?

Bye

**Comment by ivangabriele** [comment]
Hi, I don't think it's because of the version but I'm not surprised it doesn't work since I also have issues under macOS. Namely, it's not yet able to call clamav binaries within Rust command under macOS from what I tried. As for the specific TypeError you're getting, this seems strange since the types are [fully tested](https://github.com/ivangabriele/clamav-desktop/blob/main/.github/workflows/unit.yml#L22).

My plan is first to get a working version under Debian/Ubuntu with all the planned features for the v0.4.0 and then fix all the issues related to Windows and macOS. Otherwise it's really cumbersome to fix these issues one by one using either VMs or another computer.

**Comment by agala68** [comment]
I was able to overcome typescript error but I can confirm the issue to be in the call to clamav binaries, I also have tried with binaries and the seems to look for Windows paths, I have not understood at all where the app config/preferences file is. Main problem seems to be there.

**Comment by ivangabriele** [comment]
@agala68 Did you install it from a release or did you try to build it from `main`?

**Comment by agala68** [comment]
I tried both the ways, with the binaries I got errors in finding clamav bins, while with built version it seems stucked, only the scan windows was near to run, but without understanding how to make it able to read the clamv configuration.

**Comment by ivangabriele** [comment]
All that is normal then ^^. As I explained in the readme, this is only once I will have released the v0.4.0 that Clamav Desktop will be usable.
